package/MDAnalysis/core/topology.py

```diff
@@ -42,20 +42,36 @@
 def make_downshift_arrays(upshift):
     """From an upwards translation table, create the opposite direction
 
-    Turns a many to one mapping to a one to many mapping
+    Turns a many to one mapping (eg atoms to residues) to a one to many mapping
+    (residues to atoms)
 
-    Eg :: 
-      AR = np.array([0, 1, 0, 2, 2, 0, 2])
-    
-      make_downshift_arrays(AR)
-      array([array([0, 2, 5]), array([1]), array([3, 4, 6])], dtype=object)
-
-    So entry 0 informs that items 0, 2 & 5 all belong to group 0.
+    Parameters
+    ----------
+    upshift : array_like
+        Array of integers describing which parent each item belongs to
```

Comment:
The sentence is correct. The item belongs to a parent so it is the parent the item belongs to.